internals/_init.php

```
@@ -0,0 +1,231 @@
+            return 'image/gif';
+        case 'png':
+            return 'image/png';
+        case 'webp':
+            return 'image/webp';
+        default:
+            return 'false';
+    }
+}
+
+function canonicalize_cbz_dir(): string {
+    $cbz_dir = realpath(CBZ_DIR);
```

PHPMD (2.15.0)

[error] The variable $cbz_dir is not named in camelCase. (undefined)

(CamelCaseVariableName)